src/test/store/receive-profile.test.js

```diff
@@ -1759,18 +1759,35 @@ describe('actions/receive-profile', function() {
         globalTracks,
         rootRange,
       } = await setupWithLongUrl(getSomeProfiles(), {
-        urlSearch1: 'thread=0',
-        urlSearch2: 'thread=1',
+        urlSearch1: 'thread=0&profileName=name 1',
+        urlSearch2: 'thread=1&profileName=name 2',
```

Comment:
Thanks for changing a test!

What do you think of having only one of them with a `profileName`? Then we can test both possibilities :-)

Reply:
That's a good idea 💯  I've removed one of them to test the default case.